packages/sui-mono/src/spawn.js

```diff
@@ -0,0 +1,55 @@
+/* eslint no-console:0 */
```

Comment:
I think I'm missing something, but what is the difference between this and `execute.js` ?